lstchain/scripts/lstchain_dl1_to_dl2.py

```diff
@@ -106,8 +116,20 @@ def main():
     os.makedirs(args.output_dir, exist_ok=True)
     output_file = os.path.join(args.output_dir, os.path.basename(args.input_file).replace('dl1','dl2'))
 
+    if os.path.exists(output_file):
+        raise IOError(output_file + ' exists, exiting.')
+
+    dl1_keys = get_dataset_keys(args.input_file)
```

Comment:
Wouldn't it be better to define what we want to have in DL2 instead of copying and removing certain things from DL1?

Reply:
At the moment in DL1 we have:
```
 'dl1/event/subarray/mc_shower',
 'dl1/event/subarray/trigger',
 'dl1/event/telescope/image/LST_LSTCam',
 'dl1/event/telescope/parameters/LST_LSTCam',
 'dl1/event/telescope/parameters_src_dependent/LST_LSTCam',
 'instrument/subarray/layout',
 'instrument/subarray/layout.__table_column_meta__',
 'instrument/telescope/camera/ASTRICam',
 'instrument/telescope/camera/ASTRICam.__table_column_meta__',
 'instrument/telescope/camera/FlashCam',
 'instrument/telescope/camera/FlashCam.__table_column_meta__',
 'instrument/telescope/camera/LSTCam',
 'instrument/telescope/camera/LSTCam.__table_column_meta__',
 'instrument/telescope/optics',
 'instrument/telescope/optics.__table_column_meta__',
 'simulation/run_config',
 'simulation/thrown_event_distribution'
```
and we are getting rid of:
```
 'dl1/event/telescope/image/LST_LSTCam',
 'dl1/event/telescope/parameters/LST_LSTCam',
 'dl1/event/telescope/parameters_src_dependent/LST_LSTCam',
```
anything else that should not be written (this is the gamma_test file, that is why we have information from several cameras written)?

Reply:
> Wouldn't it be better to define what we want to have in DL2 instead of copying and removing certain things from DL1?

I disagree on that point. Right now the definition of the data model is going in a bottom-up approach and things are constantly added at the R1/DL0/DL1 levels. In my opinion, at the moment we should keep all the information downward.

Reply:
Sounds good.